dsls/sql/src/test/java/org/apache/beam/dsls/sql/BeamSqlDslAggregationTest.java

```diff
@@ -29,18 +29,24 @@
 import org.junit.Test;
 
 /**
- * Tests for GROUP-BY/aggregation, with global_window/fix_time_window/sliding_window/session_window.
+ * Tests for GROUP-BY/aggregation, with global_window/fix_time_window/sliding_window/session_window
+ * with BOUNDED PCollection.
  */
 public class BeamSqlDslAggregationTest extends BeamSqlDslBase {
   /**
    * GROUP-BY with single aggregation function.
    */
   @Test
   public void testAggregationWithoutWindow() throws Exception {
+    runAggregationWithoutWindow(boundedInput1);
+    runAggregationWithoutWindow(unboundedInput1);
```

Comment:
Pull these two run calls into separate test methods, please. Otherwise a failure in the bounded case means the unbounded case isn't executed (which may hide additional, different failures).

Same comment for all other similar test methods.